Client/docker-compose.yaml

```diff
@@ -22,6 +22,7 @@ services:
     network_mode: "host"
     image: gluwa/creditcoin-client:1.7
     container_name: creditcoin-client
+    read_only: true
```

Comment:
Have you tested this works? My assumption is that since the client tries to write to plugins/progress.txt when a command is run this will cause unexpected behaviour.

cc: @vladimir-gluwa

Reply:
Correct, if the client tries to write to a location that is not identified as a writable location it should cause the application to fail. 

This should be added to the documentation, all the file system locations that we are reading/writing.

Reply:
I've listed the plugins directory as tmpfs in the config file and added several more checks. However if we really need fully working containers that means we need to configure credentials inside GitHub secrets and at some point these are bound to leak. 

How do we want to handle that and keeping the repository public?

Reply:
Any deployment that requires credentials or secrets will have those secrets injected at deployment time. 
This is just so that the systems within the container cannot be modified or manipulated.